src/server/api/controllers/modules.controller.js

```diff
@@ -7,7 +7,7 @@ const moment = require('moment-timezone');
 
 const getModules = async () => {
   try {
-    return await knex('modules').select('modules.id', 'modules.title');
+    return await knex('modules').select('modules.id', 'modules.module_name');
```

Comment:
Return as `name` like you did below or as `moduleName`. Between the different endpoints, you need to be consistent.